packages/dds/map/src/interfaces.ts

```diff
@@ -18,15 +18,18 @@ export interface IValueChanged {
     /**
      * The value that was stored at the key prior to the change.
      */
+    // TODO: Use `unknown` instead (breaking change).
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     previousValue: any;
 }
 
 /**
  * Interface describing actions on a directory.
  *
- * @remarks
- * When used as a Map, operates on its keys.
+ * @remarks When used as a Map, operates on its keys.
  */
+// TODO: Use `unknown` instead (breaking change).
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export interface IDirectory extends Map<string, any>, IEventProvider<IDirectoryEvents>, Partial<IDisposable> {
```

Comment:
I'm realizing it's probably kind of weird to have the T defined on the get/set rather than here on the interface.  Can we make that change in `next` maybe? Something like
```suggestion
export interface IDirectory<T = unknown> extends Map<string, T>, IEventProvider<IDirectoryEvents>, Partial<IDisposable> {
```